lib/cli.mjs

```diff
@@ -368,10 +369,22 @@ export default async function main(argv) {
 
 	let formatter = undefined;
 
+	const cwd = process.cwd();
+	const resolvedConfig = await resolveConfig(cwd, { configFile, configBasedir });
+	const configFormatter = resolvedConfig?.formatter;
+	const cliFlagExists = Object.prototype.hasOwnProperty.call(cli.flags, 'formatter');
```

Comment:
nit

This variable name is too generic.

Reply:
which name?

Reply:
`cliFlagExists`

It could be any flag. It's really minor though.
All the code related to `formatter` being assigned should probably be refactored.
e.g. add a function
see https://github.com/stylelint/stylelint/pull/7826#discussion_r1685183481

Reply:
oh. I think I got it. changing for `cliConfigFlagExists`

Reply:
Changed.